sessionctx/variable/session.go

```diff
@@ -1691,6 +1695,8 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		s.TrackAggregateMemoryUsage = TiDBOptOn(val)
 	case TiDBMultiStatementMode:
 		s.MultiStatementMode = TiDBOptMultiStmt(val)
+	case TiDBEnableTiFlashFallbackTiKV:
```

Comment:
this variable can be protected to not let users change it. we can discuss its behavior.